src/utils/entity-utils.ts

```diff
@@ -1,20 +1,23 @@
 import { Request } from 'express';
 import { Optional } from '../typing-utils/typings';
+import { IGNORE_KEY } from '../constants';
 
 const alwaysTrueFn = (item: any) => true;
 
 export const matchEntitiesByParams = (params: Record<string, string>, paramMatch: Optional<Record<string, string>>) => {
   return paramMatch ? (entity: any) => Object.entries(params).every(([param, value]) => {
     const entityKey = paramMatch[param];
```

Comment:
what if the param isn't in the parmMatch?? Try like this `const entityKey = paramMatch[param] ?? param;`